README.md

```diff
@@ -1,14 +1,29 @@
-NYPLBase
+#NYPLBase#
```

Comment:
Needs space between # and text for format to show on GitHub, e.g. `# NYPLBase #`